cnf-certification-test/platform/cnffsdiff/fsdiff.go

```diff
@@ -46,17 +45,10 @@ type FsDiff struct {
 }
 
 func NewFsDiff(c *provider.Container) (*FsDiff, error) {
-	id := c.Status.ContainerID
-	split := strings.Split(id, "://")
-	uid := ""
-	if len(split) > 0 {
-		uid = split[len(split)-1]
-	}
-	if uid == "" {
-		logrus.Debugln(fmt.Sprintf("could not find uid of %s/%s/%s\n", c.Namespace, c.Podname, c.Data.Name))
-		return nil, errors.New("CAN'T INITIALIZE FSDIFF INSTANCE")
+	uid, err := c.GetUID()
+	if err != nil {
+		return nil, errors.New("can't instantiante FsDiff instance")
 	}
-	logrus.Debugln(fmt.Sprintf("uid of %s/%s/%s=%s\n", c.Namespace, c.Podname, c.Data.Name, uid))
 	commands := []string{"chroot", "/host", "podman", "diff", "--format", "json", uid}
```

Comment:
Why not simply merge the command here instead of RunTest?

Reply:
That was code from @hamadise's original commits.  Yeah I can edit it to just be the string.